### ISSUE TITLE
Error uploading from iOS

### ISSUE THREAD

**Issue by kamote** [balderdashy/skipper#107]
skipper version 0.5.5
upload is working from web not working from mobile app.
error
```
Error: ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms.
    at null.<anonymous> (/home/ec2-user/upnixt-api/node_modules/skipper/standalone/Upstream/Upstream.js:62:15)
    at Timer.listOnTimeout (timers.js:119:15) { [Error: ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms.]
  code: 'ETIMEOUT',
  message: 'ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms.' }
```
ios
```
NSURLSessionDataTask *task = [self.client POST:@"/upload/image" parameters:[dictionary objectForKey:@"params"]  constructingBodyWithBlock:^(id<AFMultipartFormData> formData) {
   
   NSError *error;
   NSURL *imageURL = [[dictionary objectForKey:@"params"] objectForKey:@"file"];
   [formData appendPartWithFileData:[[dictionary objectForKey:@"params"] objectForKey:@"file"] name:@"test" fileName:@"test.jpg" mimeType:@"image/jpeg"];
   //[formData appendPartWithFileURL:imageURL name:[dictionary objectForKey:@"imageFileName"] fileName:[dictionary objectForKey:@"imageFileName"] mimeType:@"image/jpeg" error:&error];

 } success:^(NSURLSessionDataTask *task, id responseObject) {
   NSLog(@"%@ - %@",responseObject, task);
 } failure:^(NSURLSessionDataTask *task, NSError *error) {
   NSLog(@"%@",error);
 }];
 NSLog(@"%@",task);
```

**Comment by sailsbot** [comment]
Thanks for posting, @kamote.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
